feature-libs/asm/root/services/asm-enabler.service.ts

```diff
@@ -26,10 +30,7 @@ export class AsmEnablerService {
     map((config: CmsConfig) => config.featureModules ?? {}),
     switchMap((featureModulesConfig) =>
       featureModulesConfig['asm']?.module
-        ? this.lazyModule.resolveModuleInstance(
-            featureModulesConfig['asm']?.module,
-            'asm'
-          )
+        ? this.featureModulesService.resolveFeature('asm')
```

Comment:
This is a new dependency, but we get to leverage/reuse the logic of lazy loading a module and it's potentially configured dependencies